### DISCUSSION TITLE
No cargo check diagnostics shows up

### DISCUSSION THREAD

**Discussion by op8867555** [mrcjkb/rustaceanvim#324] [Q&A]
Hi, I'm trying to migrate from an old rust-tools.nvim setup, but I can't get `cargo check` diagnostic works:

* All diagnostic from `cargo check` are missing, but `rust-tools.nvim` shows them correctly
  * No virtual text shows, but there is underline on error symbols
  * `:RustLsp renderDiagnostic` reports `No renderable diagnostics found`
  * `vim.diagnostic.open_float` shows nothing
* Other features works as expected: syntax diagnostic, inlay hints, ...

I've tried:
* `:checkhealth rustaceanvim` shows OK on every check
* use the minimal config setup, but no luck
* troubleshoot rust-analyzer by setting up `RA_LOG=info` but no interesting logs found

Not sure what should I do next to diagnostic this problem, please help me out

**Comment by mrcjkb** [comment]
Duplicate of https://github.com/mrcjkb/rustaceanvim/discussions/311
